federation/cluster/common.sh

```diff
@@ -123,6 +129,24 @@ function create-federated-api-objects {
 	sleep 4
     done
 
+    # Verify that federation-controller-manager pod is running.
+    for i in {1..30};do
+	#TODO(colhom): in the future this needs to scale out for N pods. This assumes just one pod
```

Comment:
We also need a generic way of expressing "wait until these pods are all running/ready", along with timeout/max retry logic